skipper.go

```diff
@@ -1468,20 +1472,13 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 
 	o.PluginDirs = append(o.PluginDirs, o.PluginDir)
 
-	var tracer ot.Tracer
-	if len(o.OpenTracing) > 0 {
-		tracer, err = tracing.InitTracer(o.OpenTracing)
-		if err != nil {
-			return err
-		}
-	} else {
-		// always have a tracer available, so filter authors can rely on the
-		// existence of a tracer
-		tracer, _ = tracing.LoadTracingPlugin(o.PluginDirs, []string{"noop"})
+	tracer, err := initializeTracer(o)
+	if err != nil {
+		return err
 	}
 
 	// tee filters override if we have a tracer
-	if len(o.OpenTracing) > 0 {
+	if tracer != nil {
```

Comment:
I think we can drop this condition since the above code ensures there is always a tracer available.

Reply:
Agreed!